container_images/daisy-builder/main.sh

```diff
@@ -20,6 +20,20 @@ ZONE="$2"
 WORKFLOW_FILE="$3"      # Workflow to run
 GCS_OUTPUT_BUCKET="$4"  # Destination for artifacts
 
+function GENERATE_NEW_VERSION() {
```

Comment:
nit: function names should be lower case

Reply:
ack